src/Core/Grid/Definition/Factory/DeleteActionTrait.php

```diff
@@ -41,25 +41,33 @@ protected function buildDeleteAction(
         string $deleteRouteParamName,
         string $deleteRouteParamField,
         string $method = 'POST',
-        array $extraRouteParams = []
+        array $extraRouteParams = [],
```

Comment:
`$extraRouteParams` cannot be included in the provided options ?
Perhaps as well for all other arguments provided in this method ?

Reply:
This is a BC Break. So I avoid it.